src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/GameFacePixelHashObserver.cs

```diff
@@ -9,13 +9,14 @@ namespace RegressionGames.StateRecorder
 {
     public class GameFacePixelHashObserver : MonoBehaviour
     {
-        private Color32[] _priorPixels;
```

Comment:
Do we need to use both `volatile` and the `Interlocked` functions to attain thread safety? IE: are the `Interlocked` functions enough?

Reply:
thanks for the clarity